indexing/INDEX_template.md

```diff
@@ -0,0 +1,33 @@
+{%  for prefix, data in content.items()%}
```

Comment:
I can't pretend to by a jinja expert, but this seems to make sense to me and it works!